src/lsp/cobol_parser/parser_outputs.ml

```diff
@@ -0,0 +1,32 @@
+(**************************************************************************)
+(*                                                                        *)
+(*                        SuperBOL OSS Studio                             *)
+(*                                                                        *)
+(*  Copyright (c) 2022-2023 OCamlPro SAS                                  *)
+(*                                                                        *)
+(* All rights reserved.                                                   *)
+(* This source code is licensed under the GNU Affero General Public       *)
+(* License version 3 found in the LICENSE.md file in the root directory   *)
+(* of this source tree.                                                   *)
+(*                                                                        *)
+(**************************************************************************)
+
+(** Gathers some types used to define outputs for the parser engine. *)
+(* We are only defining types here so an MLI would only be redundant. *)
```

Comment:
That's some kind of `types.ml` module, then ?

Reply:
Yes, but with a better name.